robottelo/ui/base.py

```diff
@@ -686,6 +686,11 @@ def select(self, target, list_value, wait_for_ajax=True, timeout=30,
         if self.element_type(target) == 'select':
             if isinstance(target, tuple):
                 element = self.wait_until_element(target)
+                if element is None:
```

Comment:
> The purpose of this PR is not to resolve the StaleElementException

The main issue with this PR is you have not reflected that in  either PR title, or in PR message, which leads to confusion. Please update commit/PR title accordingly (basically, there's no need to duplicate issue name there at all).